include/libsemigroups/present.tpp

```diff
@@ -20,6 +20,76 @@
 // monoid presentations.
 
 namespace libsemigroups {
+  namespace presentation {
+
+    template <typename T>
+    T operator<<(T const& u, T const& w) {
```

Comment:
This needs to check whether or not `T` satisfies `IsWord` as in `operator<<=` below